pkg/trace/sampler/coresampler.go

```diff
@@ -10,6 +10,7 @@ import (
 	"time"
```

Comment:
Can we rename this to `sampler.go`?

Reply:
actually `sampler.go` already exists, keeping it as it is for now